modules/persistent-storage-csi-efs-sts.adoc

```diff
@@ -4,26 +4,23 @@
 
 :_content-type: PROCEDURE
 [id="efs-sts_{context}"]
-= Configuring AWS EFS CSI Driver Operator with Security Token Service
+= Obtaining a role Amazon Resource Name for Security Token Service
 
-This procedure explains how to configure the AWS EFS CSI Driver Operator with {product-title} on AWS Security Token Service (STS).
-
-Perform this procedure before you have installed the AWS EFS CSI Operator, but not yet installed the AWS EFS CSI driver as part of the _Installing the AWS EFS CSI Driver Operator_ procedure. 
+This procedure explains how to obtain a role Amazon Resource Name (ARN) to configure the AWS EFS CSI Driver Operator with {product-title} on AWS Security Token Service (STS).
 
 [IMPORTANT]
 ====
-If you perform this procedure after installing the driver and creating volumes, your volumes will fail to mount into pods.
+Perform this procedure before you install the AWS EFS CSI Driver Operator.
 ====
 
 .Prerequisites
 
-* You have access to the cluster as a user with the cluster-admin role.
+* Access to the cluster as a user with the cluster-admin role.
 * AWS account credentials
-* You have installed the AWS EFS CSI Operator.
 
 .Procedure
 
-To configure the AWS EFS CSI Driver Operator with STS:
+To obtain a role ARN for configuring AWS EFS CSI Driver Operator using STS:
```

Comment:
It could be useful to mention that this is only one of many ways how the role can be created, and we're showing it because is uses the same concept and ccoctl tool as cluster installation.